resolwe_bio/processes/support_processors/archive.yml

```diff
@@ -6,10 +6,11 @@
     executor:
       docker:
         image: "resolwebio/archive-samples"
-  version: 0.0.2
+  version: 0.0.3
   type: "data:archive:samples"
   category: other
   persistence: TEMP
+  scheduling_class: interactive
```

Comment:
Is the archiver really an interactive process? What if there is a lot of stuff to archive?